addons/l10n_se/data/account_chart_template_post_data.xml

```diff
@@ -1,17 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data>
+        <!-- chart template -->
         <record id="l10nse_chart_template" model="account.chart.template">
             <field name="property_account_receivable_id" ref="a1510"/>
             <field name="property_account_payable_id" ref="a2440"/>
             <field name="property_account_expense_categ_id" ref="a4000"/>
             <field name="property_account_income_categ_id" ref="a3001"/>
+            <field name="complete_tax_set" eval="True"/>
+            <field name="use_anglo_saxon" eval="True"/>
+            <field name="property_stock_account_input_categ_id" ref="a1469"/>
+            <field name="property_stock_account_output_categ_id" ref="a1469"/>
+            <field name="property_stock_valuation_account_id" ref="a1460"/>
             <field name="income_currency_exchange_account_id" ref="a3960"/>
             <field name="expense_currency_exchange_account_id" ref="a3960"/>
-            <field name="property_stock_account_input_categ_id" ref="a4960"/>
-            <field name="property_stock_account_output_categ_id" ref="a4960"/>
-            <field name="property_stock_valuation_account_id" ref="a1410"/>
-            <field name="default_pos_receivable_account_id" ref="a1910"/>
-        </record> 
+            <field name="default_cash_difference_income_account_id" ref="a3740"/>
+            <field name="default_cash_difference_expense_account_id" ref="a3740"/>
+            <field name="default_pos_receivable_account_id" ref="a1510"/>
```

Comment:
This is not the default POS receivable account. Normal POS bookkeeping is usually done like this:

For CASH
1910  |  1658,00 SEK
2610  |  -240,00 SEK
2620  |  -49,07 SEK
3001  | -960,00 SEK
3002  | -408,93 SEK

For Creditcard:
1580  |  1658,00 SEK
2610  |  -240,00 SEK
2620  |  -49,07 SEK
3001  | -960,00 SEK
3002  | -408,93 SEK

When payment is registerd at bankaccount:
1580  |  -1658,00 SEK
1930  |  1658,00 SEK

1510 is used for invoices not POS. So it should be 1910 or 1580, not 1510.

Reply:
The 1580 is used for credit card. Not for cash.

The 1910 may be correct also, but its not a receivable account  and  not selectable in pos.
I don't think its in line with how odoo POS works.

in case of cash upon closing you get 2 bookings
3001 Credit
1510 Debit
And
1510 Credit
1910 Debit (If its the cash account)

You can argue its wrong, but I think is correct. 1510 is just in and out. 
The Invoice is directly payed.
You could say that odoo is issuing a "kontantfaktura" 
it's actually the same as a recipt.

You are issuing an invoice and in the same moment getting paid.

So the result would be the same as your example above.
So 1510 is not a bad option. 1910 would be more confusing.

Reply:
@jco-odoo 1910 is not a receivable account, and not correct.

Reply:
@daniel-stenlov I agree with @larmar on this one: the pos receivable account should be of type receivable and 1910 is of type cash.

Reply:
Sure..